pkg/controller/manager/manager_controller.go

```diff
@@ -309,20 +309,18 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 	}
 
 	trustedSecretNames := []string{render.PacketCaptureCertSecret, render.PrometheusTLSSecretName, relasticsearch.PublicCertSecret}
-	installCompliance := utils.IsFeatureActive(license, common.ComplianceFeature)
-	if installCompliance {
+
+	complianceLicenseFeatureActive := utils.IsFeatureActive(license, common.ComplianceFeature)
+	complianceCR, err := compliance.GetCompliance(ctx, r.client)
+	if err != nil && !errors.IsNotFound(err) {
+		r.status.SetDegraded("Error querying compliance", err.Error())
```

Comment:
Since we're touching this SetDegraded call we should update them to use the appropriate TigeraStatusReason like can be found [here](https://github.com/tigera/operator/blob/master/api/v1/tigerastatus_types.go#L120).

Reply:
Will update both SetDegraded calls - thanks for the suggestion and link.